src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+/**
+ * Formats text for display.
+ */
+public class Formatter {
+	
+	/** Offset required to convert between 1-indexing and 0-indexing.  */
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.